NODO1
------

TID: [0] [ESB] [2014-11-18 11:24:08,465]  INFO {org.wso2.carbon.application.deployer.internal.ApplicationManager} - Deploying Carbon Application : CallejeroWS_1.0.2.car... {org.wso2.carbon.application.deployer.internal.ApplicationManager}
TID: [0] [ESB] [2014-11-18 11:24:09,457]  INFO {org.wso2.carbon.application.deployer.internal.ApplicationManager} - Successfully Deployed Carbon Application : CallejeroWS {super-tenant} {org.wso2.carbon.application.deployer.internal.ApplicationManager}
TID: [0] [ESB] [2014-11-18 11:24:18,206]  INFO {org.wso2.carbon.core.deployment.CarbonDeploymentSchedulerTask} -  Sent [SynchronizeRepositoryRequest{tenantId=-1234, tenantDomain='carbon.super', messageId=822e5f0c-ee29-4b40-917d-6803405a375a}] {org.wso2.carbon.core.deployment.CarbonDeploymentSchedulerTask}
TID: [0] [ESB] [2014-11-18 11:24:33,214]  INFO {org.apache.synapse.deployers.EndpointDeployer} -  Endpoint named 'CallejeroEP' has been deployed from file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/endpoints/CallejeroEP-1.0.0.xml {org.apache.synapse.deployers.EndpointDeployer}
TID: [0] [ESB] [2014-11-18 11:24:33,216]  INFO {org.apache.synapse.core.axis2.ProxyService} -  Building Axis service for Proxy service : EXT_PUB_CallejeroWS {org.apache.synapse.core.axis2.ProxyService}
TID: [0] [ESB] [2014-11-18 11:24:33,513]  INFO {org.apache.synapse.core.axis2.ProxyService} -  Adding service EXT_PUB_CallejeroWS to the Axis2 configuration {org.apache.synapse.core.axis2.ProxyService}
TID: [0] [ESB] [2014-11-18 11:24:33,527]  INFO {org.wso2.carbon.core.deployment.DeploymentInterceptor} -  Deploying Axis2 service: EXT_PUB_CallejeroWS {super-tenant} {org.wso2.carbon.core.deployment.DeploymentInterceptor}
TID: [0] [ESB] [2014-11-18 11:24:33,772]  INFO {org.apache.synapse.core.axis2.ProxyService} -  Successfully created the Axis2 service for Proxy service : EXT_PUB_CallejeroWS {org.apache.synapse.core.axis2.ProxyService}
TID: [0] [ESB] [2014-11-18 11:24:33,773]  INFO {org.apache.synapse.deployers.ProxyServiceDeployer} -  ProxyService named 'EXT_PUB_CallejeroWS' has been deployed from file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/proxy-services/EXT_PUB_CallejeroWS-1.0.0.xml {org.apache.synapse.deployers.ProxyServiceDeployer}
TID: [0] [ESB] [2014-11-18 11:24:33,899]  INFO {org.apache.synapse.rest.API} -  Initializing API: EXT_PUB_CallejeroREST {org.apache.synapse.rest.API}
TID: [0] [ESB] [2014-11-18 11:24:33,900]  INFO {org.apache.synapse.deployers.APIDeployer} -  API named 'EXT_PUB_CallejeroREST' has been deployed from file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/api/EXT_PUB_CallejeroREST-1.0.0.xml {org.apache.synapse.deployers.APIDeployer}
TID: [0] [ESB] [2014-11-18 11:24:33,901] ERROR {org.apache.synapse.config.SynapseConfiguration} -  Duplicate entry definition for key : TratarMULTIREFCallejero {org.apache.synapse.config.SynapseConfiguration}
TID: [0] [ESB] [2014-11-18 11:24:33,902] ERROR {org.wso2.carbon.localentry.LocalEntryDeployer} -  LocalEntry Deployment from the file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/local-entries/TratarMULTIREFCallejero-1.0.0.xml : Failed. {org.wso2.carbon.localentry.LocalEntryDeployer}
org.apache.synapse.SynapseException: Duplicate entry definition for key : TratarMULTIREFCallejero
    at org.apache.synapse.config.SynapseConfiguration.handleException(SynapseConfiguration.java:1441)
    at org.apache.synapse.config.SynapseConfiguration.assertAlreadyExists(SynapseConfiguration.java:1746)
    at org.apache.synapse.config.SynapseConfiguration.addEntry(SynapseConfiguration.java:613)
    at org.apache.synapse.deployers.LocalEntryDeployer.deploySynapseArtifact(LocalEntryDeployer.java:59)
    at org.wso2.carbon.localentry.LocalEntryDeployer.deploySynapseArtifact(LocalEntryDeployer.java:43)
    at org.apache.synapse.deployers.AbstractSynapseArtifactDeployer.deploy(AbstractSynapseArtifactDeployer.java:189)
    at org.apache.axis2.deployment.repository.util.DeploymentFileData.deploy(DeploymentFileData.java:136)
    at org.apache.axis2.deployment.DeploymentEngine.doDeploy(DeploymentEngine.java:810)
    at org.apache.axis2.deployment.repository.util.WSInfoList.update(WSInfoList.java:144)
    at org.apache.axis2.deployment.RepositoryListener.update(RepositoryListener.java:377)
    at org.apache.axis2.deployment.RepositoryListener.checkServices(RepositoryListener.java:254)
    at org.apache.axis2.deployment.RepositoryListener.startListener(RepositoryListener.java:371)
    at org.apache.axis2.deployment.scheduler.SchedulerTask.checkRepository(SchedulerTask.java:59)
    at org.apache.axis2.deployment.scheduler.SchedulerTask.run(SchedulerTask.java:67)
    at org.wso2.carbon.core.deployment.CarbonDeploymentSchedulerTask.runAxisDeployment(CarbonDeploymentSchedulerTask.java:67)
    at org.wso2.carbon.core.deployment.CarbonDeploymentSchedulerTask.run(CarbonDeploymentSchedulerTask.java:112)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
    at java.util.concurrent.FutureTask$Sync.innerRunAndReset(FutureTask.java:317)
    at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:150)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$101(ScheduledThreadPoolExecutor.java:98)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.runPeriodic(ScheduledThreadPoolExecutor.java:180)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:204)
    at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
    at java.lang.Thread.run(Thread.java:662)
TID: [0] [ESB] [2014-11-18 11:24:33,903] ERROR {org.apache.synapse.deployers.AbstractSynapseArtifactDeployer} - Deployment of the Synapse Artifact from file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/local-entries/TratarMULTIREFCallejero-1.0.0.xml : Failed! {org.apache.synapse.deployers.AbstractSynapseArtifactDeployer}
org.apache.synapse.deployers.SynapseArtifactDeploymentException: LocalEntry Deployment from the file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/local-entries/TratarMULTIREFCallejero-1.0.0.xml : Failed.
    at org.apache.synapse.deployers.AbstractSynapseArtifactDeployer.handleSynapseArtifactDeploymentError(AbstractSynapseArtifactDeployer.java:466)
    at org.apache.synapse.deployers.LocalEntryDeployer.deploySynapseArtifact(LocalEntryDeployer.java:71)
    at org.wso2.carbon.localentry.LocalEntryDeployer.deploySynapseArtifact(LocalEntryDeployer.java:43)
    at org.apache.synapse.deployers.AbstractSynapseArtifactDeployer.deploy(AbstractSynapseArtifactDeployer.java:189)
    at org.apache.axis2.deployment.repository.util.DeploymentFileData.deploy(DeploymentFileData.java:136)
    at org.apache.axis2.deployment.DeploymentEngine.doDeploy(DeploymentEngine.java:810)
    at org.apache.axis2.deployment.repository.util.WSInfoList.update(WSInfoList.java:144)
    at org.apache.axis2.deployment.RepositoryListener.update(RepositoryListener.java:377)
    at org.apache.axis2.deployment.RepositoryListener.checkServices(RepositoryListener.java:254)
    at org.apache.axis2.deployment.RepositoryListener.startListener(RepositoryListener.java:371)
    at org.apache.axis2.deployment.scheduler.SchedulerTask.checkRepository(SchedulerTask.java:59)
    at org.apache.axis2.deployment.scheduler.SchedulerTask.run(SchedulerTask.java:67)
    at org.wso2.carbon.core.deployment.CarbonDeploymentSchedulerTask.runAxisDeployment(CarbonDeploymentSchedulerTask.java:67)
    at org.wso2.carbon.core.deployment.CarbonDeploymentSchedulerTask.run(CarbonDeploymentSchedulerTask.java:112)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
    at java.util.concurrent.FutureTask$Sync.innerRunAndReset(FutureTask.java:317)
    at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:150)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$101(ScheduledThreadPoolExecutor.java:98)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.runPeriodic(ScheduledThreadPoolExecutor.java:180)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:204)
    at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
    at java.lang.Thread.run(Thread.java:662)
Caused by: org.apache.synapse.SynapseException: Duplicate entry definition for key : TratarMULTIREFCallejero
    at org.apache.synapse.config.SynapseConfiguration.handleException(SynapseConfiguration.java:1441)
    at org.apache.synapse.config.SynapseConfiguration.assertAlreadyExists(SynapseConfiguration.java:1746)
    at org.apache.synapse.config.SynapseConfiguration.addEntry(SynapseConfiguration.java:613)
    at org.apache.synapse.deployers.LocalEntryDeployer.deploySynapseArtifact(LocalEntryDeployer.java:59)
    ... 21 more
TID: [0] [ESB] [2014-11-18 11:24:33,905]  INFO {org.apache.synapse.deployers.AbstractSynapseArtifactDeployer} -  The file has been backed up into : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/local-entries/TratarMULTIREFCallejero-1.0.0.xml.back {org.apache.synapse.deployers.AbstractSynapseArtifactDeployer}
TID: [0] [ESB] [2014-11-18 11:24:33,906]  INFO {org.apache.synapse.deployers.LocalEntryDeployer} -  LocalEntry named 'MULTIREF-FLAT' has been deployed from file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/local-entries/MULTIREF-FLAT-1.0.0.xml {org.apache.synapse.deployers.LocalEntryDeployer}
TID: [0] [ESB] [2014-11-18 11:24:33,911]  INFO {org.apache.synapse.deployers.SequenceDeployer} -  Sequence named 'CallejeroREST-Entrada' has been deployed from file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/sequences/CallejeroREST-Entrada-1.0.0.xml {org.apache.synapse.deployers.SequenceDeployer}
TID: [0] [ESB] [2014-11-18 11:24:33,914]  INFO {org.apache.synapse.deployers.SequenceDeployer} -  Sequence named 'CallejeroREST-Salida' has been deployed from file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/sequences/CallejeroREST-Salida-1.0.0.xml {org.apache.synapse.deployers.SequenceDeployer}
TID: [0] [ESB] [2014-11-18 11:24:33,915]  INFO {org.apache.synapse.deployers.SequenceDeployer} -  Sequence named 'ERROR-REST' has been deployed from file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/sequences/ERROR-REST-1.0.0.xml {org.apache.synapse.deployers.SequenceDeployer}
TID: [0] [ESB] [2014-11-18 11:24:49,101]  INFO {org.wso2.carbon.core.deployment.CarbonDeploymentSchedulerTask} -  Sent [SynchronizeRepositoryRequest{tenantId=-1234, tenantDomain='carbon.super', messageId=444f5238-7e81-450c-8543-6ac079e3d874}] {org.wso2.carbon.core.deployment.CarbonDeploymentSchedulerTask}



NODO2
-----

TID: [0] [ESB] [2014-11-18 11:24:15,102]  INFO {org.wso2.carbon.core.deployment.SynchronizeRepositoryRequest} - Received [SynchronizeRepositoryRequest{tenantId=-1234, tenantDomain='carbon.super', messageId=822e5f0c-ee29-4b40-917d-6803405a375a}]  {org.wso2.carbon.core.deployment.SynchronizeRepositoryRequest}
TID: [0] [ESB] [2014-11-18 11:24:18,104]  INFO {org.apache.synapse.deployers.EndpointDeployer} -  Endpoint named 'CallejeroEP' has been deployed from file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/endpoints/CallejeroEP-1.0.0.xml {org.apache.synapse.deployers.EndpointDeployer}
TID: [0] [ESB] [2014-11-18 11:24:18,192]  INFO {org.apache.synapse.rest.API} -  Initializing API: EXT_PUB_CallejeroREST {org.apache.synapse.rest.API}
TID: [0] [ESB] [2014-11-18 11:24:18,193]  INFO {org.apache.synapse.deployers.APIDeployer} -  API named 'EXT_PUB_CallejeroREST' has been deployed from file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/api/EXT_PUB_CallejeroREST-1.0.0.xml {org.apache.synapse.deployers.APIDeployer}
TID: [0] [ESB] [2014-11-18 11:24:18,194] ERROR {org.apache.synapse.config.SynapseConfiguration} -  Duplicate entry definition for key : TratarMULTIREFCallejero {org.apache.synapse.config.SynapseConfiguration}
TID: [0] [ESB] [2014-11-18 11:24:18,195] ERROR {org.wso2.carbon.localentry.LocalEntryDeployer} -  LocalEntry Deployment from the file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/local-entries/TratarMULTIREFCallejero-1.0.0.xml : Failed. {org.wso2.carbon.localentry.LocalEntryDeployer}
org.apache.synapse.SynapseException: Duplicate entry definition for key : TratarMULTIREFCallejero
    at org.apache.synapse.config.SynapseConfiguration.handleException(SynapseConfiguration.java:1441)
    at org.apache.synapse.config.SynapseConfiguration.assertAlreadyExists(SynapseConfiguration.java:1746)
    at org.apache.synapse.config.SynapseConfiguration.addEntry(SynapseConfiguration.java:613)
    at org.apache.synapse.deployers.LocalEntryDeployer.deploySynapseArtifact(LocalEntryDeployer.java:59)
    at org.wso2.carbon.localentry.LocalEntryDeployer.deploySynapseArtifact(LocalEntryDeployer.java:43)
    at org.apache.synapse.deployers.AbstractSynapseArtifactDeployer.deploy(AbstractSynapseArtifactDeployer.java:189)
    at org.apache.axis2.deployment.repository.util.DeploymentFileData.deploy(DeploymentFileData.java:136)
    at org.apache.axis2.deployment.DeploymentEngine.doDeploy(DeploymentEngine.java:810)
    at org.apache.axis2.deployment.repository.util.WSInfoList.update(WSInfoList.java:144)
    at org.apache.axis2.deployment.RepositoryListener.update(RepositoryListener.java:377)
    at org.apache.axis2.deployment.RepositoryListener.checkServices(RepositoryListener.java:254)
    at org.apache.axis2.deployment.RepositoryListener.startListener(RepositoryListener.java:371)
    at org.apache.axis2.deployment.scheduler.SchedulerTask.checkRepository(SchedulerTask.java:59)
    at org.apache.axis2.deployment.scheduler.SchedulerTask.run(SchedulerTask.java:67)
    at org.wso2.carbon.core.deployment.CarbonDeploymentSchedulerTask.runAxisDeployment(CarbonDeploymentSchedulerTask.java:67)
    at org.wso2.carbon.core.CarbonAxisConfigurator.runDeployment(CarbonAxisConfigurator.java:419)
    at org.wso2.carbon.core.deployment.SynchronizeRepositoryRequest.doDeployment(SynchronizeRepositoryRequest.java:84)
    at org.wso2.carbon.core.deployment.SynchronizeRepositoryRequest.execute(SynchronizeRepositoryRequest.java:73)
    at org.apache.axis2.clustering.tribes.RpcMessagingHandler.replyRequest(RpcMessagingHandler.java:55)
    at org.apache.catalina.tribes.group.RpcChannel.messageReceived(RpcChannel.java:134)
    at org.apache.axis2.clustering.tribes.Axis2GroupChannel.messageReceived(Axis2GroupChannel.java:79)
    at org.apache.catalina.tribes.group.ChannelInterceptorBase.messageReceived(ChannelInterceptorBase.java:84)
    at org.apache.catalina.tribes.group.interceptors.NonBlockingCoordinator.messageReceived(NonBlockingCoordinator.java:512)
    at org.apache.catalina.tribes.group.ChannelInterceptorBase.messageReceived(ChannelInterceptorBase.java:84)
    at org.apache.catalina.tribes.group.interceptors.TcpFailureDetector.messageReceived(TcpFailureDetector.java:113)
    at org.apache.catalina.tribes.group.ChannelInterceptorBase.messageReceived(ChannelInterceptorBase.java:84)
    at org.apache.catalina.tribes.group.interceptors.DomainFilterInterceptor.messageReceived(DomainFilterInterceptor.java:45)
    at org.apache.catalina.tribes.group.ChannelInterceptorBase.messageReceived(ChannelInterceptorBase.java:84)
    at org.apache.catalina.tribes.group.interceptors.OrderInterceptor.messageReceived(OrderInterceptor.java:101)
    at org.apache.catalina.tribes.group.ChannelInterceptorBase.messageReceived(ChannelInterceptorBase.java:84)
    at org.apache.catalina.tribes.group.ChannelCoordinator.messageReceived(ChannelCoordinator.java:253)
    at org.apache.catalina.tribes.transport.ReceiverBase.messageDataReceived(ReceiverBase.java:287)
    at org.apache.catalina.tribes.transport.nio.NioReplicationTask.drainChannel(NioReplicationTask.java:212)
    at org.apache.catalina.tribes.transport.nio.NioReplicationTask.run(NioReplicationTask.java:101)
    at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
    at java.lang.Thread.run(Thread.java:662)
TID: [0] [ESB] [2014-11-18 11:24:18,196] ERROR {org.apache.synapse.deployers.AbstractSynapseArtifactDeployer} - Deployment of the Synapse Artifact from file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/local-entries/TratarMULTIREFCallejero-1.0.0.xml : Failed! {org.apache.synapse.deployers.AbstractSynapseArtifactDeployer}
org.apache.synapse.deployers.SynapseArtifactDeploymentException: LocalEntry Deployment from the file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/local-entries/TratarMULTIREFCallejero-1.0.0.xml : Failed.
    at org.apache.synapse.deployers.AbstractSynapseArtifactDeployer.handleSynapseArtifactDeploymentError(AbstractSynapseArtifactDeployer.java:466)
    at org.apache.synapse.deployers.LocalEntryDeployer.deploySynapseArtifact(LocalEntryDeployer.java:71)
    at org.wso2.carbon.localentry.LocalEntryDeployer.deploySynapseArtifact(LocalEntryDeployer.java:43)
    at org.apache.synapse.deployers.AbstractSynapseArtifactDeployer.deploy(AbstractSynapseArtifactDeployer.java:189)
    at org.apache.axis2.deployment.repository.util.DeploymentFileData.deploy(DeploymentFileData.java:136)
    at org.apache.axis2.deployment.DeploymentEngine.doDeploy(DeploymentEngine.java:810)
    at org.apache.axis2.deployment.repository.util.WSInfoList.update(WSInfoList.java:144)
    at org.apache.axis2.deployment.RepositoryListener.update(RepositoryListener.java:377)
    at org.apache.axis2.deployment.RepositoryListener.checkServices(RepositoryListener.java:254)
    at org.apache.axis2.deployment.RepositoryListener.startListener(RepositoryListener.java:371)
    at org.apache.axis2.deployment.scheduler.SchedulerTask.checkRepository(SchedulerTask.java:59)
    at org.apache.axis2.deployment.scheduler.SchedulerTask.run(SchedulerTask.java:67)
    at org.wso2.carbon.core.deployment.CarbonDeploymentSchedulerTask.runAxisDeployment(CarbonDeploymentSchedulerTask.java:67)
    at org.wso2.carbon.core.CarbonAxisConfigurator.runDeployment(CarbonAxisConfigurator.java:419)
    at org.wso2.carbon.core.deployment.SynchronizeRepositoryRequest.doDeployment(SynchronizeRepositoryRequest.java:84)
    at org.wso2.carbon.core.deployment.SynchronizeRepositoryRequest.execute(SynchronizeRepositoryRequest.java:73)
    at org.apache.axis2.clustering.tribes.RpcMessagingHandler.replyRequest(RpcMessagingHandler.java:55)
    at org.apache.catalina.tribes.group.RpcChannel.messageReceived(RpcChannel.java:134)
    at org.apache.axis2.clustering.tribes.Axis2GroupChannel.messageReceived(Axis2GroupChannel.java:79)
    at org.apache.catalina.tribes.group.ChannelInterceptorBase.messageReceived(ChannelInterceptorBase.java:84)
    at org.apache.catalina.tribes.group.interceptors.NonBlockingCoordinator.messageReceived(NonBlockingCoordinator.java:512)
    at org.apache.catalina.tribes.group.ChannelInterceptorBase.messageReceived(ChannelInterceptorBase.java:84)
    at org.apache.catalina.tribes.group.interceptors.TcpFailureDetector.messageReceived(TcpFailureDetector.java:113)
    at org.apache.catalina.tribes.group.ChannelInterceptorBase.messageReceived(ChannelInterceptorBase.java:84)
    at org.apache.catalina.tribes.group.interceptors.DomainFilterInterceptor.messageReceived(DomainFilterInterceptor.java:45)
    at org.apache.catalina.tribes.group.ChannelInterceptorBase.messageReceived(ChannelInterceptorBase.java:84)
    at org.apache.catalina.tribes.group.interceptors.OrderInterceptor.messageReceived(OrderInterceptor.java:101)
    at org.apache.catalina.tribes.group.ChannelInterceptorBase.messageReceived(ChannelInterceptorBase.java:84)
    at org.apache.catalina.tribes.group.ChannelCoordinator.messageReceived(ChannelCoordinator.java:253)
    at org.apache.catalina.tribes.transport.ReceiverBase.messageDataReceived(ReceiverBase.java:287)
    at org.apache.catalina.tribes.transport.nio.NioReplicationTask.drainChannel(NioReplicationTask.java:212)
    at org.apache.catalina.tribes.transport.nio.NioReplicationTask.run(NioReplicationTask.java:101)
    at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
    at java.lang.Thread.run(Thread.java:662)
Caused by: org.apache.synapse.SynapseException: Duplicate entry definition for key : TratarMULTIREFCallejero
    at org.apache.synapse.config.SynapseConfiguration.handleException(SynapseConfiguration.java:1441)
    at org.apache.synapse.config.SynapseConfiguration.assertAlreadyExists(SynapseConfiguration.java:1746)
    at org.apache.synapse.config.SynapseConfiguration.addEntry(SynapseConfiguration.java:613)
    at org.apache.synapse.deployers.LocalEntryDeployer.deploySynapseArtifact(LocalEntryDeployer.java:59)
    ... 33 more
TID: [0] [ESB] [2014-11-18 11:24:18,197]  INFO {org.apache.synapse.deployers.AbstractSynapseArtifactDeployer} -  The file has been backed up into : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/local-entries/TratarMULTIREFCallejero-1.0.0.xml.back {org.apache.synapse.deployers.AbstractSynapseArtifactDeployer}
TID: [0] [ESB] [2014-11-18 11:24:18,198]  INFO {org.apache.synapse.deployers.LocalEntryDeployer} -  LocalEntry named 'MULTIREF-FLAT' has been deployed from file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/local-entries/MULTIREF-FLAT-1.0.0.xml {org.apache.synapse.deployers.LocalEntryDeployer}
TID: [0] [ESB] [2014-11-18 11:24:18,201]  INFO {org.apache.synapse.deployers.SequenceDeployer} -  Sequence named 'CallejeroREST-Entrada' has been deployed from file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/sequences/CallejeroREST-Entrada-1.0.0.xml {org.apache.synapse.deployers.SequenceDeployer}
TID: [0] [ESB] [2014-11-18 11:24:18,204]  INFO {org.apache.synapse.deployers.SequenceDeployer} -  Sequence named 'CallejeroREST-Salida' has been deployed from file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/sequences/CallejeroREST-Salida-1.0.0.xml {org.apache.synapse.deployers.SequenceDeployer}
TID: [0] [ESB] [2014-11-18 11:24:18,205]  INFO {org.apache.synapse.deployers.SequenceDeployer} -  Sequence named 'ERROR-REST' has been deployed from file : /usr/local/platina/wso2esb-4.7.0/repository/deployment/server/synapse-configs/default/sequences/ERROR-REST-1.0.0.xml {org.apache.synapse.deployers.SequenceDeployer}

TID: [0] [ESB] [2014-11-18 11:24:39,102]  INFO {org.wso2.carbon.core.deployment.SynchronizeRepositoryRequest} - Received [SynchronizeRepositoryRequest{tenantId=-1234, tenantDomain='carbon.super', messageId=444f5238-7e81-450c-8543-6ac079e3d874}]  {org.wso2.carbon.core.deployment.SynchronizeRepositoryRequest}
TID: [0] [ESB] [2014-11-18 11:24:41,100]  INFO {org.wso2.carbon.deployment.synchronizer.subversion.SVNBasedArtifactRepository} -  Unversioned file problem. Retrying 1 {org.wso2.carbon.deployment.synchronizer.subversion.SVNBasedArtifactRepository}
TID: [0] [ESB] [2014-11-18 11:24:48,204]  INFO {org.wso2.carbon.deployment.synchronizer.subversion.SVNBasedArtifactRepository} -  Unversioned file: /usr/local/platina/esb/repository/deployment/server/synapse-configs/default/local-entries/TratarMULTIREFCallejero-1.0.0.xml.back will be deleted {org.wso2.carbon.deployment.synchronizer.subversion.SVNBasedArtifactRepository} 
